### PR TITLE
Remove userId auth option

### PR DESCRIPTION
**Pull request by daniyarchambylov**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -27,16 +26,11 @@ class Client {
...
+      throw new TypeError('Please provide auth options.')
     }
 
-    if (password && !ethAddress && !apiKey) {
-      throw new TypeError('Please provide an user id auth option.')
-    }
-
-    if (!apiKey && !userId && (!password && ethAddress)) {
+    if (!apiKey && (!password || !ethAddress)) {
       throw new TypeError('Please provide a password auth option.')
```
Comment (fgimenez) [inline index.js]: With these changes i think the error message is wrong now, maybe `Please provide either ethAddress and password or apiKey`, this is actually what we are checking now, WDYT?

**Review by fgimenez — DISMISSED** [review]
Looks good, only a small nitpick.

File: index.js (hunk 1 above)
Comment (daniyarchambylov) [reply index.js]: Sounds good to me.

Comment (daniyarchambylov) [reply index.js]: Done!

**Review by fgimenez — APPROVED** [review]
Great thx!

**Review by rocky — APPROVED** [review]
LGTM

**Pull request merged**